src/BuildScriptGenerator/Python/PythonBashBuildSnippet.sh.tpl

```diff
@@ -125,9 +125,9 @@ fi
         echo
         echo Running pip install...
         START_TIME=$SECONDS
-        InstallCommand="$python -m pip install --cache-dir $PIP_CACHE_DIR --prefer-binary -r $REQUIREMENTS_TXT_FILE --target="{{ PackagesDirectory }}" --upgrade | ts $TS_FMT"
+        InstallCommand="$python -m pip install --cache-dir $PIP_CACHE_DIR --prefer-binary -r $REQUIREMENTS_TXT_FILE --target="{{ PackagesDirectory }}" {{ oryxPipUpgradeCommand }} | ts $TS_FMT"
```

Comment:
The two `oryxPipUpgradeCommand` references that you've added in this file need to be changed to match the property name in the `PythonBashBuildSnippetProperties`, which is currently `OryxPipCommand` (or `PipUpgradeFlag` if we change the property name), so that they are properly rendered when the script is generated

Reply:
This comment still applies -- the value of `PipUpgradeFlag` in the `PythonBashBuildSnippetProperties` won't be rendered in this template file if the property name in the template file does not match.